src/modules/pos/validator/components/StakeSummary/StakeSummary.test.js

```diff
@@ -176,6 +182,22 @@ describe('StakingQueue.Summary', () => {
     expect(wrapper).toContainMatchingElements(12, '.stake-item-address');
   });
 
+  it('should render rewards', () => {
+    const wrapper = mountWithRouter(Summary, { ...props, edited, removed, added });
+    const addedItemList = wrapper.find('.stake-item').at(0);
+    const editedItemList = wrapper.find('.stake-item').at(4);
+    const removedItemList = wrapper.find('.stake-item').at(9);
+    expect(addedItemList).toHaveText(
+      'lskdwsyfmcko6mcd357446yatromr9vzgu7eb8y110 LSKReward:  0.001 LSK'
```

Comment:
This is wrong ? it should always display validator name? i.e `subzero 100 LSK Reward: 0.001 LSK`?

Reply:
reason was because in the mock prop value, the validator name wasn't provided it was just the address.